object_store/src/aws/client.rs

```
@@ -669,8 +672,13 @@ impl S3Client {
             request = request.with_encryption_headers();
         }
         let response = request.send().await?;
+        let checksum = response
+            .headers()
+            .get(SHA256_CHECKSUM)
+            .and_then(|v| v.to_str().ok())
```

Do we need to unwind these errors or swallow them?